remoteit/desktop
# DESK-1130 not launching

### Description


Fixing recursion of application token updates DESK-1130
Fixing onboarding guide appearance bug

### Test plan



1. Sign in
2. Go to a connection
3. Ensure you have missing token data
4. Click launch
5. It should prompt for missing data
6. After data is entered it should launch with that data filled out

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1130>
